.github/workflows/gh-ci-pulsar-tests.yml

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
"pulsar" may be ambigous, maybe we can use "pulsar-sink".

We will have two separate and totally independent topics:
- Pulsar Sink: this is a Pulsar IO component to write to Venice
- Using Pulsar instead of Kafka: this is about the internals of Venice

Reply:
I added `[pulsar-sink]` tag to the PR also added to the templates.